flutter_local_notifications/lib/src/flutter_local_notifications_plugin.dart

```diff
@@ -118,6 +119,22 @@ class FlutterLocalNotificationsPlugin {
     if (kIsWeb) {
       return true;
     }
+    if (Platform.isAndroid && initializationSettings.android == null) {
+      throw ArgumentError(
+          'Android settings must be set when targeting Android platform');
+    }
+    if (Platform.isIOS && initializationSettings.iOS == null) {
+      throw ArgumentError(
+          'iOS settings must be set when targeting iOS platform');
+    }
+    if (Platform.isLinux && initializationSettings.linux == null) {
+      throw ArgumentError(
+          'Linux settings must be set when targeting Linux platform');
+    }
+    if (Platform.isMacOS && initializationSettings.macOS == null) {
+      throw ArgumentError(
+          'MacOS settings must be set when targeting MacOS platform');
```

Comment:
casing issue here the platform itself is referred to as "macOS" with a lowercase "m"

Reply:
The first MacOs is the beginning of a sentence. As far I know English grammatic, you must start a sentence with a Capital letter.